tests/test_remote_git.py

```diff
@@ -65,7 +65,7 @@ def test_non_ascii_tags():
 
 def test_make_force_push_mutate_refs_func_overwrites_shas():
     targets = ['refs/heads/targeta', 'refs/tags/targetb']
-    newsha = 'newsha'
+    newsha = b'newsha'
```

Comment:
this constant isn't terribly useful now (it's probably more readable to not have the constant)